Kitodo/src/main/java/org/kitodo/services/file/FileService.java

```diff
@@ -241,15 +285,17 @@ public Integer getNumberOfFiles(URI directory) {
      */
     public Integer getNumberOfImageFiles(URI directory) {
         int count = 0;
-        if (isDirectory(directory)) {
-            /*
-             * die Images zählen
-             */
+        if (directory.isAbsolute()) {
+            if (isDirectory(directory)) {
+                count = getSubUris(Helper.imageNameFilter, directory).size();
```

Comment:
Why did you add the number of subdirectories as a start value for the number of image files? Looks a bit strange.

Like in method `getNumberOfFiles` both code lines of the `if`statement looks similar and could be separated.